subworkflows/local/vcf_chunk_glimpse/vcf_chunk_glimpse.nf

```diff
@@ -0,0 +1,72 @@
+include { GLIMPSE_CHUNK                     } from '../../../modules/nf-core/glimpse/chunk/main'
+include { GLIMPSE2_CHUNK                    } from '../../../modules/nf-core/glimpse2/chunk/main'
+include { GLIMPSE2_SPLITREFERENCE           } from '../../../modules/nf-core/glimpse2/splitreference/main'
+
+workflow VCF_CHUNK_GLIMPSE {
+
+    take:
+    ch_reference                           // channel: [ val(meta),vcf ]
+    ch_map                                 // channel  (optional): [ meta, map ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    // Add chromosome to channel
+    ch_vcf_csi_chr = ch_reference.map{meta, vcf, csi -> [meta, vcf, csi, meta.chr]}
+
+    // Make chunks with Glimpse1
+    GLIMPSE_CHUNK(ch_vcf_csi_chr)
```

Comment:
Is it necessary to process twice the panel ?
Maybe we should stick to only one algorithm ?

Reply:
I was thinking the same, but the chunking results are different. I am not sure if you could use the chunking files from `glimpse2` into `glimpse1` imputation. I think we could try to optimize it when we use the external param `--chunks`. We could take a glimpse2 output and modify it to use the required sections necessary for `glimpse1` imputation. I'll add it as an issue!